docs/conf.py

```diff
@@ -136,6 +136,7 @@ def documenter(app, obj, parent):
     'sphinx.ext.coverage',
     'sphinx.ext.napoleon',
     'sphinx_multiversion',
+    'sphinx.ext.autosectionlabel',
```

Comment:
needed to make references by section name/label as done in tutorial 4 for 'Random Number Generation'.